frontend/public/components/catalog-items.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import * as classNames from 'classnames';
+
+import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
+import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
+import {normalizeIconClass} from './catalog-item-icon';
+
+const CatalogItem = ({item}) => {
+  const catalogTile = <CatalogTile
+    id={item.obj.metadata.uid}
+    key={item.obj.metadata.uid}
+    title={item.tileName}
+    iconImg={item.tileImgUrl}
+    iconClass={item.tileIconClass ? `icon ${normalizeIconClass(item.tileIconClass)}` : null}
+    vendor={item.tileProvider ? `Provided by ${item.tileProvider}` : null}
+    description={item.tileDescription}
+  />;
+  return item.href ? <a href={item.href}>{catalogTile}</a> : catalogTile;
```

Comment:
This actually need to be a `<Link to={item.href}>` otherwise the entire app reloads

(I know I said `<a>`, my fault.)